packages/account/src/Sections/Profile/FinancialAssessment/financial-assessment.jsx

```diff
@@ -216,7 +217,14 @@ class FinancialAssessment extends React.Component {
                 setStatus({ msg: data.error.message });
             } else {
                 this.setState({ is_submit_success: true });
-                setTimeout(() => this.setState({ is_submit_success: false }), 3000);
+                setTimeout(
+                    () =>
+                        this.setState({
+                            is_submit_success: false,
+                            has_completed_financial_assessment: true,
```

Comment:
can this be moved to line 219? why does it need to be in the timeout?

Reply:
Moving it to line 219 outside of the timeout will cause the green ticked button of 'is_submit_success' state to be greyed out as well during the transition. I don't think that's the intended behaviour, but do correct me if I'm wrong.
Being in the timeout, will allow for allow for the green ticked button to appear fully (not greyed out) followed by transitioning to the greyed out button, as attached:

![FA_Button](https://user-images.githubusercontent.com/72253841/95408097-5354cf00-0951-11eb-9420-a72a9864bcdc.gif)